app/assets/stylesheets/modules/forms.scss

```diff
@@ -3,8 +3,19 @@
     padding-top: 7px;
   }
 
+  hr {
+    margin-top: 20px;
+    margin-bottom: 20px;
+    border: 0;
+    border-top: 1px solid #d5d5d3;
```

Comment:
Color literals like `#d5d5d3` should only be used in variable declarations; they should be referred to via variable everywhere else.